mono/mini/aot-compiler.c

```diff
@@ -7509,7 +7544,7 @@ compile_method (MonoAotCompile *acfg, MonoMethod *method)
 		if (acfg->aot_opts.print_skipped_methods)
 			printf ("Skip (abs call): %s\n", mono_method_get_full_name (method));
 		InterlockedIncrement (&acfg->stats.abscount);
-		mono_destroy_compile (cfg);
+		pending_compile_destroy (acfg, cfg);
 		return;
```

Comment:
Why is this needed ? Keeping cfg-s around will dramatically increase the memory usage of the aot compiler.